modules/virt-importing-vm-datavolume.adoc

```diff
@@ -74,12 +75,12 @@ spec:
             storage: 2Gi
         storageClassName: local
       source:
-        http:
-          url: https://download.fedoraproject.org/pub/fedora/linux/releases/28/Cloud/x86_64/images/Fedora-Cloud-Base-28-1.1.x86_64.qcow2 <1>
-          secretRef: "" <2>
-          certConfigMap: "" <3>
-    status: {}
-  running: false
+        http: <1> 
```

Comment:
Do we want an example using the registry source?

Reply:
I didn't want to stack examples in the config itself to the callouts for this and the next line have the registry alternatives:
<1> The source type from which to import the image. This example uses a HTTP endpoint. To import a container disk from a registry, replace `http` with `registry`.
<2> The source of the virtual machine image you want to import. This example references a virtual machine image at an HTTP endpoint. An example of a container registry endpoint is `url: "docker://kubevirt/fedora-cloud-container-disk-demo:latest"`.

If that's not clear enough, I could change L78 to "http:|registry:" and adjust the callout to match.